fdbclient/BackupContainerFileSystem.actor.cpp

```diff
@@ -1133,16 +1133,17 @@ class BackupContainerFileSystemImpl {
 		    filename,
 		    IAsyncFile::OPEN_ATOMIC_WRITE_AND_CREATE | IAsyncFile::OPEN_READWRITE | IAsyncFile::OPEN_CREATE,
 		    0600));
-		StreamCipher::Key::RawKeyType testKey;
-		generateRandomData(testKey.data(), testKey.size());
+		StreamCipherKey testKey(AES_256_KEY_LENGTH);
+		testKey.initializeRandomTestKey();
+		// generateRandomData(testKey.data(), testKey.size());
```

Comment:
```suggestion
```

Reply:
Ahh.. Seems like dead code; thanks for pointing it out.